lib/README.md

```diff
@@ -0,0 +1,7 @@
+## Queue Interface
+
+Enqueing a message is done via a call to sendMessage.
+
+Messages without or with a "send_after" property in the past, are sent immediately.
+
+Messages with a "send_after" property in the future are stored in the database with a status of 'delayed'. These messages can be sent by a call to sendDelayedMessages().
```

Comment:
Also we do things `camelCase` everywhere else so I think `send_after` should be `sendAfter`